mcp/mastra/verticals/coding/workflows.ts

```diff
@@ -178,22 +178,33 @@ const prepareIssueCreationData = createStep({
   }),
   execute: async (params) => {
     const state = params.state;
-    const { requirements } = state.response;
+    const requirements = state.response?.requirements;
+
+    if (!requirements) {
+      throw new Error('No requirements found in workflow state');
+    }
 
     // Format requirements as markdown
-    const requirementsSection = requirements.requirements.map((req) => `- ${req}`).join('\n');
-    const acceptanceCriteriaSection = requirements.acceptanceCriteria.map((ac) => `- [ ] ${ac}`).join('\n');
+    const reqList = requirements.requirements ?? [];
```

Comment:
Don't enshorten variable names. Quite the contrary. This should always apply, so add that to the global agents.md for the whole repo.